### PR TITLE
Added PerDimWeightedAverage component for chain models in SWBD, works…

### PR DESCRIPTION
**Pull request by vijayaditya**
… as good as jesus (tdnn_4f) model

This is a re-submission of PR #461

### CODE REVIEW

**Comment by danpovey** [comment]
You added egs/wsj/s5/steps/nnet3/tdnn/train.sh.  How does this differ from
nnet3/train_tdnn.sh and would there now be duplication?

**Comment by vijayaditya** [comment]
There is duplication. I would ideally just keep
steps/nnet3/tdnn/{train.sh,make_configs.py} as it is similar in directory
structure to steps/nnet3/lstm/{train.sh,makes_configs.py}.

I will gradually change all existing local/* scripts to use
steps/nnet3/tdnn/* scripts and then delete steps/nnet3/train_tdnn.sh script
and the corresponding make_tdnn_configs.py scripts.

File: egs/swbd/s5c/local/chain/run_tdnn_4q.sh [hunk 1]
```diff
@@ -0,0 +1,181 @@
+#!/bin/bash
+
+# this is based on Dan's tdnn_2o script
+# it has a different splicing configuration
+# it uses the PerDimensionWeightedAverage pooling in place of the Jesus layer
+
+# it relies on new steps/nnet3/chain/train_tdnn_b.sh script which accepts more
```
Comment (danpovey) [inline egs/swbd/s5c/local/chain/run_tdnn_4q.sh]: the script doesn't have the _b name now.

Comment (vijayaditya) [inline egs/swbd/s5c/local/chain/run_tdnn_4q.sh]: oops.. changing  it

**Comment by danpovey** [comment]
already merged, include it next time you make a change.

**Comment by vijayaditya** [comment]
OK.

**Pull request merged**